vnext/PropertySheets/React.Cpp.props

```diff
@@ -16,6 +16,8 @@
     <CharacterSet>Unicode</CharacterSet>
   </PropertyGroup>
 
+  <Import Project="$(ReactNativeWindowsDir)PropertySheets\ReactPatches.targets" />
```

Comment:
Per the recent comment, going to keep this in the props file for now. If we end up needing more targets, we can make a breaking change to add a shared targets file but I'm not a fan of doing this until we have more than just this target. This is especially true since we have static properties coupled to the target that I would like to ideally keep together.